src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,42 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health) {
+    this.name = name;
+    this.health = health || 100;
+    Animal.alive.push(this);
+  }
+
+  dead() {
+    const index = Animal.alive.indexOf(this);
+
+    Animal.alive.splice(index, 1);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+  }
+
+  hide(hidden) {
+    this.hidden = true;
+  }
 }
 
 class Carnivore extends Animal {
```

Comment:
where is constructor in Carnivore class?

Reply:
constructor(name) {
    super(name);
  }

Linter says it's 'useless constructor'. What shoud I add here?